src/components/global/utils/egoJwt.ts

```diff
@@ -0,0 +1,37 @@
+import jwtDecode from 'jwt-decode';
+import { memoize } from 'lodash';
+import jwt from 'jsonwebtoken';
+
+import { EGO_PUBLIC_KEY } from 'common/injectGlobals';
+import { EgoJwtData, User } from 'common/typedefs';
+
+const verifyJwt: (egoPublicKey: string) => (egoJwt?: string) => boolean = (egoPublicKey) => (
+  egoJwt,
+) => {
+  try {
+    if (!egoJwt || !egoPublicKey) {
+      return false;
+    } else {
+      return jwt.verify(egoJwt, egoPublicKey, { algorithms: ['RS256'] }) && true;
```

Comment:
Not sure why we have `&& true`... if the intention is to force boolean return type, maybe we want `!!jwt.verify(...)`